README.md

```diff
@@ -43,10 +43,12 @@ $ bundle install
 $ yarn install
 $ rake dev:setup
 $ heroku local -f Procfile.dev
+# if you chose the local route, then you are good to go on:
+  http://localhost:5000
```

Comment:
@smoline if I use `rails s` (and not `rails s -p 5000`), it's still expecting 3000. Just wondering if 5000 is working for you? Also, if the local route is chosen, I think we do need a line above the localhost reference w the relevant rails s.